docs/images/bluefin/framework.md

```diff
@@ -2,6 +2,35 @@ Bluefin is available as an image for the Framework 13 laptop that comes preconfi
 
 Note that the default image works fine on the Framework 13, this image provides tweaks and further improvements. Additionally if you have power profiles that you think would be useful for the community please send a pull request! 
 
+## Changes
+
+Implements the changes from Framework's [Optimizing Fedora Battery Life](https://knowledgebase.frame.work/en_us/optimizing-fedora-battery-life-r1baXZh) knowledgebase article:
+
+- Replaces `power-profiles-daemon` with `tlp`
+- Ships the [recommended power profile](https://github.com/ublue-os/bluefin/blob/main/framework/etc/tlp.d/50-framework.conf) for `tlp`
+- Sets the text scaling factor in GNOME to [1.25](https://github.com/ublue-os/bluefin/blob/main/framework/etc/dconf/db/local.d/01-ublue-framework)
+  - Since Bluefin comes with fractional scaling already enabled by default you may want to adjust those settings, however this will case Electron apps to become blurry. Setting the text font larger seems to be the least worst solution we could come up with.
```

Comment:
...this will [cause] Electron apps to become blurry.

suggest "least worst" in quotations.